code/Control/UserDefinedFormController.php

```diff
@@ -52,6 +52,13 @@ class UserDefinedFormController extends PageController
     /** @var string The name of the folder where form submissions will be placed by default */
     private static $form_submissions_folder = 'Form-submissions';
 
+    /**
+     * Size in megabytes that an uploaded file must not excede for it to be attached to an email
+     * @var int
+     * @config
+     */
+    private static $maximum_email_attachment_size = 1;
```

Comment:
Should this be a bit more granular, e.g. using kilobytes instead of megabytes?

Reply:
I don't think it necessary, I can't imagine a situation where 2.7MB is OK, but 3MB is not.
Remember this is controlling whether an attachment is set on an email, not whether or not an upload is accepted via the userform :)

Reply:
Perhaps reflect that in the name then? `maximum_email_attachment_size_mb` ?

Reply:
Any movement on this?

Reply:
@NightJar would you kindly add the `_mb` into the config name and then we can accept this? Thanks!

Reply:
It would be great to have this arbitrary value configurable. Some ideas for options:

+ maximum_email_attachment_size_mb = 0 - flag that no attachments are able to be added
+ maximum_email_attachment_size_mb = null - flag that all attachments are able to be added (no max limit defined)

Reply:
@MrJamesEllis that would be in the implementation of how the method uses the config, the default would need to stay as `1` for backward compatibility.

I'd say we can accept the PR and then enhance on it.

Reply:
@NightJar keen on adding the `_mb` suffix so we can accept this? Thanks!

Reply:
Nah, not really eh.